resip/dum/InviteSession.cxx

```diff
@@ -2161,9 +2175,17 @@ InviteSession::dispatchOthers(const SipMessage& msg)
       case MESSAGE:
          dispatchMessage(msg);
          break;
-	  case ACK:
-		  // Ignore duplicate ACKs from 2xx reTransmissions
-		  break;
+      case ACK:
+         // Not checking for strict equality, since request may have been digest challenged
+         if (mLastRemoteSessionModification->header(h_CSeq).sequence() > msg.header(h_CSeq).sequence())
+         {
+            InfoLog(<< "dropped stale ACK");
+         }
+         else
+         {
+            mCurrentRetransmit200 = 0; // stop the 200 retransmit timer
+         }
```

Comment:
Is there a reason you are not calling:
handler->onAckReceived(getSessionHandle(), msg);
here?

Reply:
I was trying to do the minimum in this default method, but I will revert the change from the last commit to make it in line with the code in dispatchSentReinvite.

Reply:
I think it's good to leave the code in dispatchOthers - but I think it also makes sense to call onAckReceived  :)

Reply:
Now calling onAckRecieived from dispatchOthers.